### PR TITLE
Contract Directory Restructure

### PR DESCRIPTION
**Pull request by devpavan04**
### Description

- This PR restructures the contract directory by making Safes, Claims and Guardians as separate contracts which are inherited by the SafientMain contract.

### Changed

- `SafientMain` contract inherits `Safes`, `Claims` and `Guardians` contracts along with `IArbitrable` interface, making `SafientMain` as the arbitrable contract.
- `recoverSafeFunds` function name is changed to `retrieveSafeFunds` in the contract and tests.
- `inheritor` is changed to `beneficiary` in the contract and tests.

### Modified

- There exists two test files, `contractTest.js` and `SDKTest.js`. `contractTest.js` is included in the `test` directory, `SDKTest.js` is included in the `root` directory for running both the tests together. Needs further modifications.

### Removed

- `safeId` attribute is removed from `Claims` struct.
- `getClaimsOnSafeBySafeId` function is removed from SDK since `safeId` attribute is removed from `Claims` struct.

### CODE REVIEW

File: contracts/contracts/Guardians.sol [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
```
Comment (koshikraj) [inline contracts/contracts/Guardians.sol:1]: Let's rename the inner `contracts` to `components`.

File: package.json [hunk 2]
```diff
@@ -14,7 +14,9 @@
     "clean": "hardhat clean",
```
Comment (koshikraj) [inline package.json:14]: Update the version to `0.1.3`.

File: contracts/contracts/Claims.sol [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        evidenceGroupID += 1;
+
+        emit Events.Dispute(
+            data.arbitrator,
+            disputeID,
+            data.metaEvidenceId,
+            evidenceGroupID
+        );
+
+        Types.Claim memory claim = claims[disputeID];
```
[the remark this answers is not in the document]
Comment (devpavan04) [reply contracts/contracts/Claims.sol:102]: Yes. We are creating a new claim with the returned disputeId and updating that claim in the claims mapping.

[the remark this answers is not in the document]
Comment (koshikraj) [reply contracts/contracts/Claims.sol:102]: But this is a new claim, right? So the `claims[disputeID]` will have a null

**Comment by koshikraj** [comment]
LGTM

**Pull request merged**